codespell_lib/data/dictionary.txt

```diff
@@ -4281,6 +4293,7 @@ eigth->eighth, eight,
 eihter->either
 einstance->instance
 eiter->either
+either either->either
```

Comment:
I thought that we were unable to find such errors. Does this actually work? If not, we should add a check in the dict checking tests that there is no whitespace in the dict keys

Reply:
My experience with spellintian is that it's a bit of a nightmare of false positives anyway. It should probably just be added as another option which does duplicate word checking, but spellintian does it across line boundaries so it catches things like fi twice in bash scripts, but I suspect line boundaries are the most likely place for people to make duplicate word entries, so it's lose lose (which ironically is an intentional duplication).

Reply:
sorry, so should I remove this? BTW.. will codespell still catch typos like this ?

Reply:
No it won't in its current form. Eventually it might be worth adding such two-word combos but someone would have to volunteer and be motivated to do it.

Reply:
Well, said person should then remember to re-implement 
https://github.com/codespell-project/codespell/pull/616/files#diff-3bddc99f1f6dffc98ea11be5a5a31ae9

Reply:
Indeed, as noted in the description of that PR

Reply:
Although I'd say yes to the related words, but not the duplicates stripped out of that. However I imagine the default, and custom regexes, are looking for words, so it's more a case of comparing the current and following word and then probably only suggesting not auto-replacing. E.g. my regex matches underscore separated variable names.

Reply:
P.S. This still needs removing to fix the test @luzpaz .